keylime/api_version.py

```diff
@@ -23,21 +24,21 @@ def latest_minor_version(v):
     return "0"
 
 
-def all_versions():
+def all_versions() -> List[str]:
     return VERSIONS.copy()
 
 
-def is_supported_version(v):
+def is_supported_version(v: str) -> bool:
     v_obj = version.parse(str(v))
     return v_obj.base_version in VERSIONS
 
 
-def is_deprecated_version(v):
+def is_deprecated_version(v: str) -> bool:
     v_obj = version.parse(str(v))
     return is_supported_version(v) and v_obj.base_version in DEPRECATED_VERSIONS
 
 
-def normalize_version(v):
+def normalize_version(v: str) -> str:
     v = str(v)
```

Comment:
here, and later there are more. I vote to remove all of them and see what is broken